app/design/adminhtml/Magento/backend/Magento_Catalog/web/css/source/_module.less

```diff
@@ -9,6 +9,7 @@
 
 .catalog-product-index {
     .admin__data-grid-wrap {
+        overflow-x: scroll;
         .data-row {
```

Comment:
in myopinion properties should overflow-x: auto instead always show scroll

Reply:
@adamphh This commit only fixes product grid but this issue is happening for all the magento backend ui-grids.
@mrtuvn

Reply:
@irajneeshgupta Thanks for your comment. I will resolve it